js/base/Exchange.js

```diff
@@ -555,6 +556,10 @@ module.exports = class Exchange {
         }
     }
 
+    exchangeSpecificJsonAlterations (jsonString) {
```

Comment:
@samgermain can you please double-check if we really need this?

Reply:
I remember there being no other way around it, I tested it a lot when I originally wrote the PR, that string conversion is needed on NDAX, but it felt like a waste to do it on every exchange

Reply:
@samgermain is it possible to share your keys with me so I can test this on my side?